src/Symfony/Component/DependencyInjection/Extension/Extension.php

```diff
@@ -82,13 +81,8 @@ public function getConfiguration(array $config, ContainerBuilder $container)
         $namespace = $reflected->getNamespaceName();
 
         $class = $namespace.'\\Configuration';
-        if (class_exists($class)) {
-            $r = new \ReflectionClass($class);
-            $container->addResource(new FileResource($r->getFileName()));
```

Comment:
Fair enough. I still think the "change" is considerable, but we can keep things as is here. Doing things twice by default.